config/routes.rb

```diff
@@ -51,6 +51,12 @@
   get "/register", to: "users#new"
   post "/users", to: "users#create"
   get "/profile", to: "users#show"
+
```

Comment:
The combo of user and profile routes seems very unrestful, would love any suggestions.

Reply:
seems good to me *shrug*